tests/manage/monitoring/test_workload_with_distruptions.py

```diff
@@ -172,3 +178,168 @@ def test_workload_with_checksum_verify(
     ocp_pod = ocp.OCP(kind="Pod", namespace=project.namespace)
     sha1sum_output = ocp_pod.exec_oc_cmd(f"logs {pod_name}", out_yaml_format=False)
     logger.info("sha1sum output: %s", sha1sum_output)
+
+
+def get_mon_pod_by_pvc_name(pvc_name: str):
+    """
+    Function to get monitor pod by pvc_name label
+
+    Args:
+        pvc_name (str): name of the pvc the monitor pod is related to
+    """
+    mon_pod_ocp = (
+        ocp.OCP(
+            kind=constants.POD,
+            namespace=config.ENV_DATA["cluster_namespace"],
+            selector=f"pvc_name={pvc_name}",
+        )
+        .get()
+        .get("items")[0]
+    )
+    return Pod(**mon_pod_ocp)
+
+
+class TestCephOSDSlowOps(object):
+    @pytest.fixture(scope="function")
+    def setup(self, request, pod_factory, multi_pvc_factory):
+        """
+        Set preconditions to trigger CephOSDSlowOps
+        """
+        self.test_pass = None
+        reduced_osd_complaint_time = 0.1
+
+        def set_osd_op_complaint_time(osd_op_complaint_time_val: float):
+            self.ct_pod = pod.get_ceph_tools_pod()
+            self.ct_pod.exec_ceph_cmd(
+                f"ceph config set osd osd_op_complaint_time {osd_op_complaint_time_val}"
+            )
```

Comment:
this should be available for other tests, could be placed in https://github.com/red-hat-storage/ocs-ci/blob/master/ocs_ci/ocs/cluster.py

Reply:
addressed in https://github.com/red-hat-storage/ocs-ci/pull/8345 and cherry-picked